src/PowerShellEditorServices/Services/TextDocument/Handlers/FormattingHandlers.cs

```diff
@@ -14,17 +14,24 @@
 
 namespace Microsoft.PowerShell.EditorServices.Handlers
 {
-    internal class DocumentFormattingHandler : IDocumentFormattingHandler
+    // TODO: Add IDocumentOnTypeFormatHandler to support on-type formatting.
+    internal class DocumentFormattingHandlers : IDocumentFormattingHandler, IDocumentRangeFormattingHandler
     {
         private readonly ILogger _logger;
         private readonly AnalysisService _analysisService;
         private readonly ConfigurationService _configurationService;
         private readonly WorkspaceService _workspaceService;
-        private DocumentFormattingCapability _capability;
 
-        public DocumentFormattingHandler(ILoggerFactory factory, AnalysisService analysisService, ConfigurationService configurationService, WorkspaceService workspaceService)
+        private DocumentFormattingCapability _documentFormattingCapability;
+        private DocumentRangeFormattingCapability _documentRangeFormattingCapability;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_documentRangeFormattingCapability' or refactor the code to use its value.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5162794)